velox/common/memory/MemoryArbitrator.cpp

```diff
@@ -243,6 +244,17 @@ void MemoryReclaimer::abort(MemoryPool* pool, const std::exception_ptr& error) {
   });
 }
 
+bool MemoryReclaimer::Stats::operator==(
+    const MemoryReclaimer::Stats& other) const {
+  return std::tie(numNonReclaimableAttempts) ==
```

Comment:
Don't need for std::tie as there is only one element

Reply:
For later extension. But yeah we can just compare the single variable for now.